database-week1/database-week1.sql

```diff
@@ -0,0 +1,31 @@
+-- Active: 1724726987336@@127.0.0.1@3306@hyf_lesson1
+-- 1 task. Selects all tasks from task table
+SELECT COUNT(*) FROM task -- 35 tasks
+
+-- 2 task. Selects all tasks from task table where due_date is null
+SELECT COUNT(*) FROM task WHERE due_date IS NULL -- 8 tasks
+
+-- 3 task. Selects all tasks from task table where status_id is 3(Done)
+SELECT * FROM task WHERE status_id = 3 -- 12 tasks
```

Comment:
you should be able to do this query by looking for the title instead of the id as well

Reply:
Thank you for the suggestion, i have made the changes.